src/editor/StyleEditor.cpp

```diff
@@ -74,6 +76,20 @@ void tryLoadPalette(const QString &basedir, const QString &styleName) {
   }
 }
 
+void initializeStyleDir() {
+  QString optimalLocation =
+      QStandardPaths::writableLocation(QStandardPaths::AppLocalDataLocation) +
+      "/styles/";
+
+  QFile readme(":/styles/README.md");
+  QDir outputDir(optimalLocation);
+  if (!outputDir.exists(optimalLocation)) {
+    outputDir.mkpath(optimalLocation);
+  }
+  readme.remove();
```

Comment:
why are we removing this? what does it even mean to remove a file from the qrc?

Reply:
oh yeah i meant to actually remove that from the destination, idk if QFile::remove can be used as a static method but if so then it 's an easy fix